<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>canvasTab</class>
 <widget class="QWidget" name="canvasTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="plotFrame2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="plotFrame">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="plotLayout">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>679</width>
        <height>259</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="plotLayout2">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="indexFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_head">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_pre">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_page">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>23</height>
         </size>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_page">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_all">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>/共__</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_next">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tail">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_to">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>当前最多存储20个标记点，跳转时请注意！</string>
        </property>
        <property name="text">
         <string>跳转</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
